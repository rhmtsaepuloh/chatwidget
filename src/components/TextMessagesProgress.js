import React from "react";
import { MdSchedule } from "react-icons/md";

const TextMessagesProgress = ({ message, timestamp }) => {
  return (
    <React.Fragment>
      <p className='m-1 whitespace-pre-wrap break-words'>{message}</p>
      <div className='file-timestamp flex items-center justify-end'>
        <p className='text-sm' style={{ color: "#979797" }}>
          {timestamp}
        </p>
        <MdSchedule className='ml-1' />
      </div>
    </React.Fragment>
  );
};

export default TextMessagesProgress;
