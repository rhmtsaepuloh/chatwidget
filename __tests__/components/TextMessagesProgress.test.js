/* eslint-disable react/react-in-jsx-scope */
import React from "react";
import { shallow } from "enzyme";
import TextMessagesProgress from "../../src/components/TextMessagesProgress";

const props = {
  message: "cek",
  timestamp: "10:21",
};

describe("render correctly text messages progress components", () => {
  test("renders", () => {
    const wrapper = shallow(<TextMessagesProgress {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
