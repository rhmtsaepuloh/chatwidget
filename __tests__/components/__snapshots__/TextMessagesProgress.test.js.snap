// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly text messages progress components renders 1`] = `ShallowWrapper {}`;
